trunk/NDHMS/Routing/module_HYDRO_io.F

```diff
@@ -7395,6 +7395,7 @@ subroutine READ_CHROUTING1( &
 integer, intent(IN), dimension(IXRT,JXRT)    :: CH_NETLNK
 real,  dimension(IXRT,JXRT)                  :: ChSSlpG,BwG,TwG,MannNG  !channel properties
 real,  dimension(IXRT,JXRT)                  :: Tw_CCG,n_CCG            !channel properties of compound
+real,  dimension(IXRT,JXRT)                  :: ChannKG                 !Channel Infiltration
```

Comment:
I added ChannK here, but I don't think channel loss is supported with the gridded channel. I am fine keeping this variable here as a placeholder for future development though.

Reply:
okay, lets keep it. at some point I assume we'll add for gridded.